x-pack/plugins/actions/server/builtin_action_types/lib/connector_token_client.test.ts

```diff
@@ -375,6 +416,7 @@ describe('updateOrReplace()', () => {
       connectorId: '1',
       token: null,
       newToken: 'newToken',
+      tokenRequestDate: Date.now(),
```

Comment:
I am curious, how do the test are working with different dates on each run?

Reply:
Good catch! It looks like the existing unit tests are testing just whether certain values are passed to `create` and exclude testing the date. I will add a test to ensure that `tokenRequestDate` is used when it's passed in.

Reply:
Thanks!